src/js/network-vis.js

```diff
@@ -1,9 +1,12 @@
 import { Network } from 'vis-network/peer/esm/vis-network';
 import { DataSet } from 'vis-data/peer/esm/vis-data';
 import { colors } from './constants';
+import 'jstree/dist/themes/default/style.css';
```

Comment:
jstree dependencies and all jstree logic should not exist in this component

Reply:
Agree. Needs a complete restructure of the code; else there will be a performance hit iterating through 10s of thousands of data blocks multiple times. I can take this tech debt now.